en/docs/consume/manage-subscription.md

```diff
@@ -1,23 +1,32 @@
-# Subscribe to an API
+# Test an API
+ 
+Before using your API in production, you can test it using the integrated console in Choreo. Once you deploy your API in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
 
-Subscribing an API to an application allows Choreo to authenticate the APIs requests with the application keys. You have to subscribe to a **published** API to an application to invoke it using the application credentials.
+This guide will take you through the steps to test an API in Choreo. 
 
-Alternatively, you can generate credentials for an API without an explicit subscription to an application. However, this will not let you control advanced configuration such as access token expiry time, revoke token expiry time, ID token expiry time, and enabling access to the API without a secret. Generating keys in the API is recommended for testing or short-term usage but not for long-term production usage. 
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click **Developer Portal**  on the Choreo Console header. 
 
-This guide takes you through the steps to subscribe to APIs in Choreo. 
+    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
 
-Let's get started!
+2. Click **APIs**.
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click on the Developer Portal link on the Choreo console header. 
+3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alternatively, you can click on the API and follow step 4. 
+4. In the left pane, click **Try Out**.
+5. Select the required environment from the **Environment** list.
 
-    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}
+    ![Test Panel](../assets/img/developer-portal/test-api/test-panel.png){.cInlineImage-half}
+
+6. From the **Subscribed Application** list, select the application you used to subsribe the API to. To learn how to subscribe to an application, see [Manage Subscriptions](../manage-subscription/). 
```

Comment:
The application subscribes to the API and not the other way around isn't it?

Shall we change this as follows?

"From the **Subscribed Application** list, select the application that needs to subscribe to the API. To learn how an application can subscribe to an API, see [Manage Subscriptions](../manage-subscription/)."